app/build.gradle

```
@@ -46,6 +46,7 @@ android {
             } else {
                 applicationIdSuffix ".debug." + normalizedWorkingBranch
                 resValue "string", "app_name", "NewPipe " + workingBranch
+                archivesBaseName = 'NewPipe_' + workingBranch
```

Better to use `normalizedWorkingBranch`, as `workingBranch` could contain special characters not allowed in filenames.
```suggestion
                archivesBaseName = 'NewPipe_' + normalizedWorkingBranch
```